types/index.d.ts

```diff
@@ -67,17 +76,17 @@ declare class MockAdapter {
 
   history: { [method: string]: AxiosRequestConfig[] };
 
-  onGet: RequestMatcherFunc;
+  onList: NoBodyRequestMatcherFunc;
+  onOptions: NoBodyRequestMatcherFunc;
+  onAny: NoBodyRequestMatcherFunc;
+  onLink: NoBodyRequestMatcherFunc;
+  onUnlink: NoBodyRequestMatcherFunc;
```

Comment:
I believe the types for `link` and `unlink` are not correct. Since they are not included in `methodsWithConfigsAsSecondArg`, they are expected to have the configs as third arguments (which I believe is correct from what I've found: https://datatracker.ietf.org/doc/html/draft-snell-link-method).

For the `list` method, I haven't found any resources and it also doesn't seem to be implemented by Axios, so I wonder if we should remove it.

Reply:
Thanks. I guess link and unlink are coming from webdav.
I guess we can ignore list. 🙃